Decred Wallet/Extensions/UIButton.swift

```diff
@@ -16,14 +16,13 @@ extension UIButton {
         case bottom
     }
     
-    
-    func addBorders(atPositions borderPositions: [BorderPosition], colorHex: String = "#333333", thickness: CGFloat = 1.7) {
+    func addBorders(atPositions borderPositions: [BorderPosition], colorHex: String = "#4e5f70", thickness: CGFloat = 1.7) {
         borderPositions.forEach({ borderPosition in
             self.addBorder(atPosition: borderPosition, colorHex: colorHex, thickness: thickness)
         })
     }
     
-    func addBorder(atPosition borderPosition: BorderPosition, colorHex: String = "#333333", thickness: CGFloat = 1.7) {
+    func addBorder(atPosition borderPosition: BorderPosition, colorHex: String = "#4e5f70", thickness: CGFloat = 1.7) {
```

Comment:
For consistency could we inject in the Color object that we can can define in our `UICOLOR extensions` file instead of passing in the colorHex.